security/aws/loadbalancer_internet_facing/aws_internet-facing_elbs.pt

```diff
@@ -262,19 +272,16 @@ end
 ###############################################################################
 
 escalation "report_internet_facing_ELB" do
+  automatic true
+  label "Send Mail"
+  description "Sends incidents email"
   email $param_email
 end
 
 escalation "approve_delete_internet_facing_ELB" do
-  request_approval  do
-    label "Approve Resource Deletion"
-    description "Approve escalation to run RightScale Cloud Workflow to delete Internet-facing ELB's & ALB's"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "Explain why you are approving the action"
-    end
-  end
+  automatic false
+  label "Delete Internet-facing ELB's & ALB's"
```

Comment:
Change to Delete Resource, or Delete ELB

Reply:
Done !